hbase10/README.md

```diff
@@ -49,39 +49,36 @@ hbase(main):002:0> create 'usertable', 'cf', {SPLITS => (1..n_splits).map {|i| "
 
 Make a note of the column family, in this example it's `cf``.
 
-### 4. Fetch the Proper ALPN Boot Jar
+### 4. Download the Bigtable Client Jar with required dependencies:
```

Comment:
For folks who download the all-of-YCSB artifact, aren't these jars already in the directory for the 'googlebigtable' binding?